packages/back-end/src/services/licenseData.ts

```diff
@@ -64,24 +61,9 @@ export async function getLicenseMetaData() {
 }
 
 export async function initializeLicenseForOrg(
-  reqOrg?: OrganizationInterface,
+  org?: OrganizationInterface,
   forceRefresh = false
 ) {
-  let org = reqOrg;
-  if (!org) {
-    // When we initialize the app, we don't have a request with an org yet.
-    if (IS_CLOUD) {
-      // On cloud we will initialize license on an as need basis once we know the org of a request
-      return;
-    }
-
-    // For self-hosted there is either
-    // no orgs upon first initialization - in which case we get the key from the env var
-    // exactly one org normally - in which case we get the key from the org
-    // or multiple orgs if it is a MULTI_ORG site - in which case the org is not allowed to have a license key and we shall also fall back to the env var
-    org = (await getOrganization()) || undefined;
```

Comment:
I didn't know this `getOrganization()` method existed.  We should delete this ASAP as it is extremely dangerous to have in the codebase.  It doesn't check for self-hosted, so if this were to be called on cloud, it could potentially expose a random organization to other users.

Reply:
Good point